docs/compute-resources.md

```diff
@@ -63,7 +63,8 @@ Each of these details is explained in more depth below.
 Some points to note:
 
 - Task-level resource requests and limits do not apply to sidecars which can be configured separately.
-- Users may not configure the Task-level and Step-level resource requirements (requests/limits)  simultaneously.
+- Resource requirements configured in `Step` or `StepTemplate` of the referenced `Task` will be overridden by the task-level requirements.
```

Comment:
Also, please update line 48 to clarify that this feature is implemented!

Reply:
yap.